menas/ui/components/run/runMain.view.xml

```diff
@@ -0,0 +1,189 @@
+<!--
+  ~ Copyright 2018-2019 ABSA Group Limited
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<core:View id="runMainView" xmlns:core="sap.ui.core" xmlns="sap.m"
+           controllerName="components.run.runMain"
+           xmlns:form="sap.ui.layout.form"
+           xmlns:table="sap.ui.table"
+           xmlns:cust="http://schemas.sap.com/sapui5/extension/sap.ui.core.CustomData/1">
+    <Page title="Runs" enableScrolling="false">
+        <customHeader>
+            <Bar>
+                <contentMiddle>
+                    <Title text="Runs"/>
+                </contentMiddle>
+                <contentRight>
+                    <Text text="Logged in as:"/>
```

Comment:
Was there a fragment for these added recently?

Reply:
Thanks for spotting that! Oversight on my part.